app/views/omniauth_callbacks/login.html.erb

```diff
@@ -9,6 +9,10 @@
   </div>
 
   <div class="login-form">
-    <%= button_to 'Sign in with Google', '/auth/google_oauth2', class: 'google-signin-btn' %>
+    <% if ENV['GOOGLE_CLIENT_ID'] %>
+      <%= button_to 'Sign in with Google', '/auth/google_oauth2', class: 'signin-btn google' %>
+    <% else %>
+      <%= button_to 'Sign in with SAML', '/auth/saml', class: 'signin-btn okta' %>
+    <% end %>
```

Comment:
Do we want to if/else here? Or should we just allow both if the settings are there?